src/components/themed/ThemedButtons.tsx

```diff
@@ -92,7 +92,7 @@ export function RightChevronButton(props: { text: string; onPress: () => void; p
   const padding = sidesToPadding(mapSides(fixSides(paddingRem, 0), theme.rem))
 
   return (
-    <TouchableOpacity onPress={onPress}>
+    <TouchableOpacity accessible={false} onPress={onPress}>
```

Comment:
What children were made accessible here?

Reply:
The EdgeText is made accessible